packages/react-scripts/scripts/start.js

```diff
@@ -152,6 +152,12 @@ checkBrowsers(paths.appPath, isInteractive)
         process.exit();
       });
     });
+
+    process.stdin.on('end', function() {
+      devServer.close();
+      process.exit();
```

Comment:
Just wondering if this is a successful run, why not use `process.exit(0)` here?

Reply:
Hi @konekoya. According to the node docs:

> If code is omitted, exit uses either the 'success' code 0 or the value of process.exitCode if it has been set. 

I'm not sure if process.exitCode has been set anywhere else. Making it explicit sounds like a good idea. I didn't write this solution so I'm not sure what the original thought process was.

Reply:
Thanks for the detailed explanation :)